lib/toy/object.rb

```diff
@@ -32,11 +32,11 @@ def class
     end
 
     def new
-      Instance.new(self)
+      ObjectInstance.new(self)
```

Comment:
Another option is to put these instance classes into a `Toy::Instance` namespace, i.e. define `Toy::Instance::Object` in `lib/toy/instance/object.rb` and so on. I don’t know whether that’s clearer or not!

Reply:
My brain wants to keep `ObjectInstance` for now, sorry Tom 😂  I suppose I will eventually be persuaded as I was with moving things to the `Toy::` namespace though haha